### PR TITLE
Feature clear results

### LINKED ISSUE #3: Clean results support
screen recording 2018-02-20 at 10 22 pm

### PR DESCRIPTION
**Pull request by breadadams**
Close #3

### CODE REVIEW

**Comment by Kikobeats** [comment]
OMG! 😍 

some points before merge

At Medium they are adding a bit of delay after hide the Close button when the mouse is out of the component.

screen recording 2018-03-05 at 09 50 am

The Background color of the Close Icon should be use `primaryColor` and `secondaryColor` to be possible use custom color properly.

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "mo-js": "~0.288.2",
     "polished": "~1.9.2",
     "react-transition-group": "^2.2.1",
-    "styled-components": "~3.1.4"
+    "styled-components": "~3.2.0"
```
Comment (Kikobeats) [inline package.json]: careful because the current `3.2.0` is actually broken!

File: src/components/ClapButton.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const ClapButton = styled.button`
   position: relative;
   outline: 1px solid transparent;
   border-radius: 50%;
-  background: none;
+  background: #fff;
```
Comment (Kikobeats) [inline src/components/ClapButton.js:22]: maybe are you thinking in `transparent`?

Comment (breadadams) [reply src/components/ClapButton.js:22]: I had to set a background because you could see the ✖️ behind the circle

**Pull request merged**